flow/emulator/emulator.go

```diff
@@ -37,37 +33,33 @@ var Cmd = &cobra.Command{
 }
 
 func configuredServiceKey(
-	init bool,
-	sigAlgo crypto.SignatureAlgorithm,
-	hashAlgo crypto.HashAlgorithm,
+	_ bool,
+	_ crypto.SignatureAlgorithm,
+	_ crypto.HashAlgorithm,
 ) (
 	crypto.PrivateKey,
 	crypto.SignatureAlgorithm,
 	crypto.HashAlgorithm,
 ) {
-	if sigAlgo == crypto.UnknownSignatureAlgorithm {
-		sigAlgo = emulator.DefaultServiceKeySigAlgo
-	}
-
-	if hashAlgo == crypto.UnknownHashAlgorithm {
-		hashAlgo = emulator.DefaultServiceKeyHashAlgo
-	}
+	proj := cli.LoadProject()
 
-	var serviceAcct *cli.Account
+	serviceAccount := proj.EmulatorServiceAccount()
 
-	if init {
-		pconf := initialize.InitProject(sigAlgo, hashAlgo)
-		serviceAcct = pconf.ServiceAccount()
+	serviceKey := serviceAccount.Keys[0]
 
-		fmt.Printf("⚙️   Flow client initialized with service account:\n\n")
-		fmt.Printf("👤  Address: 0x%s\n", serviceAcct.Address)
-	} else {
-		serviceAcct = cli.LoadConfig().ServiceAccount()
+	privateKey, err := crypto.DecodePrivateKeyHex(serviceKey.SigAlgo, serviceKey.Context["privateKey"])
```

Comment:
Hmm, this is a bit brittle. 

I guess _only_ hex keys can be used as the service account key, right?

I also realized that `proj.EmulatorServiceAccount()` returns type `config.Account`, but really it should return [this Account type](https://github.com/onflow/flow-cli/blob/master/flow/project/cli/project.go#L237-L242), right?

That way you could do this:

```
serviceKeyHex, ok := serviceKey.(*keys.HexAccountKey)
if !ok {
  cli.Exit(1, "Only hexadecimal keys can be used as the emulator service account key."
}

privateKey := serviceKeyHex.PrivateKey()
```

This could require you to add this function to `keys.HexAccountKey`:

```go
func (a *HexAccountKey) PrivateKey() crypto.PrivateKey {
	return a.privateKey
}